### ISSUE TITLE
Resolve "FutureWarning" message perhaps via a different version of numpy

### ISSUE THREAD

**Issue by njtierney** [greta-dev/greta#402]
Currently in [tests](https://github.com/greta-dev/greta/runs/2497322018?check_suite_focus=true#step:15:69) and during interactive use, greta will display something like:

``` 
/home/rstudio/.local/share/r-miniconda/envs/r-reticulate/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
```

According to this [SO thread](https://stackoverflow.com/questions/57381430/synonym-of-type-is-deprecated-in-a-future-version-of-numpy-it-will-be-underst) this is apparently due to a compatibility issue with the version of numpy and tensorflow.

**Comment by njtierney** [comment]
looks like [according to this SO thread](https://stackoverflow.com/questions/58662410/which-numpy-versions-are-compatible-with-tensorflow-1-14-0) I should try out numpy 1.16.4

**Comment by njtierney** [comment]
should be able to resolve with

```r
install_tensorflow(
  method = "conda",
  version = "1.14.0",
  extra_packages = c("tensorflow-probability==0.7.0", "numpy==16.4")
)
```

**Comment by njtierney** [comment]
@goldingn do you think we should detect the version of numpy used? It will still work with newer versions but it gives these verbose warnings that might put people off?

**Comment by goldingn** [comment]
I think specifying the particular version of numpy to install is the best approach. Especially if we can automate that with the rest of the python package installation. We could remove the version dependency later, when we remove the dependency on this version of TF. The warnings should disappear in later TF versions.

**Comment by njtierney** [comment]
ok cool - we can add installation instructions but I'm not sure the best way to detect the current version of `numpy` as I think my approach in https://github.com/rstudio/reticulate/issues/1000 initialises python and I don't want to do that I don't think?

**Comment by goldingn** [comment]
Do we need to detect the version at all? It's not an issue if they use a later version, so long as TF works.

**Comment by njtierney** [comment]
Alrighty awesome, i'll add a line to the suggested installation code and leave it at that.

**Comment by njtierney** [comment]
resolved in #412
